doc/source/docs/reference/geodataframe.rst

```diff
@@ -4,20 +4,20 @@ GeoDataFrame
 .. currentmodule:: geopandas
 
 A ``GeoDataFrame`` is a tabular data structure that contains a column
-called ``geometry`` which contains a ``GeoSeries``.
+which contains a ``GeoSeries`` storing geometry.
 
 Constructor
 -----------
 .. autosummary::
-   :toctree: api/
+   :toctree: geodataframe/
```

Comment:
you might have changed this because I asked about "api" being "reference", but wondering: Is the additional sub-directory needed? 

Eg the docstring page of GeoSeries is now at "/docs/reference/geoseries/geopandas.GeoSeries.html", but this could maybe also be "docs/reference/geopandas.GeoSeries.isna.html"

Or would it become annoying that the generated rst pages get mixed in the same directory as the non-generated overview pages like this one? 

(the different subdirectories (geoseries, tools, io, testing) might also get confusing to get right. If a subdir is needed, a single one for all doctring pages might be sufficient?)

Reply:
> Or would it become annoying that the generated rst pages get mixed in the same directory as the non-generated overview pages like this one?

Exactly right, that is why I put them in sub-folders. 

> the different subdirectories (geoseries, tools, io, testing) might also get confusing to get right. If a subdir is needed, a single one for all doctring pages might be sufficient?

Might be, this just seemed better structured. Happy to change that though.

We can have `/docs/reference/api/geopandas.GeoSeries.html"`, exactly as pandas have (although there's still a difference of homepage + sphinx and only sphinx we have.).

Reply:
> We can have /docs/reference/api/geopandas.GeoSeries.html", exactly as pandas have (

That would be fine with me (but can also be done as follow-up)